platformtour/public/css/platformtour.css

```diff
@@ -138,8 +138,8 @@ tr.introjs-showElement > th {
   position: absolute;
   padding: 10px;
   background-color: white;
-  min-width: 200px;
-  max-width: 300px;
+  min-width: 300px;
```

Comment:
Why do we need to bump the `min-width` and not just `max-width`?

Reply:
`min-width` is actually the necessary one, but I increased `max-width` as well to keep the available range...

Reply:
I'm not sure I follow..

Isn't the range increasing from `100px` (`300 - 200`) to `200px` (`500 - 300`),
while increasing in total possible width by `200px`?

If we're just tweaking the `min-width`, why do we also need to let it get bigger than it used to in other cases?

Reply:
whoops, I meant for max to `400px`